Workspaces/MiniWorld/Scripts/MiniWorldRenderer.cs

```diff
@@ -53,7 +53,7 @@ void OnEnable()
 		{
 			var go = new GameObject("MiniWorldCamera", typeof(Camera));
```

Comment:
Should use ObjectUtils.Instantiate. I think at the time I was thinking we'd have this work in play mode, but that's a much bigger task to make everything work now.